core-tests/src/test/java/org/mule/runtime/core/privileged/event/PrivilegedEventTestCase.java

```diff
@@ -89,7 +88,7 @@ public void removeLoggingVariableOnEventWithThatVariableCreatesNewEvent() throws
   public void removeLoggingVariable() throws MuleException {
     CoreEvent originalEvent = builder(testEvent()).addLoggingVariable("some", "value").build();
     PrivilegedEvent newEvent = builder(originalEvent).removeLoggingVariable("some").build();
-    assertThat(newEvent.getLoggingVariables().isPresent(), is(true));
-    assertThat(newEvent.getLoggingVariables().get().containsKey("some"), is(false));
+    assertThat(newEvent.getLoggingVariables(), is(not(empty())));
+    newEvent.getLoggingVariables().ifPresent(vars -> assertThat(vars.keySet(), not(contains("some"))));
```

Comment:
This could have stayed as before, since in the previous line you asserted that it was present, but okey 🤷

Reply:
We don't like Sonar warnings